test/run-tests

```diff
@@ -32,14 +33,24 @@ COMMENT_CMD_TEST_ALL = "[citest]"
 COMMENT_CMD_TEST_PENDING = "[citest pending]"
 COMMENT_CMD_TEST_BAD = "[citest bad]"
 
-TRANSIENT_ERROR_WAIT_TIME_DEFAULT = 30
-TRANSIENT_ERROR_WAIT_TIME_MAX = 600
-transient_error_wait_time = TRANSIENT_ERROR_WAIT_TIME_DEFAULT
 
+class transient_error_wait_time:
```

Comment:
Classues use `CamelCase` by convention in Python, for example `TransientErrorWaitTime`.